compiler/record-minmax/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+nnas_find_package(Boost REQUIRED COMPONENTS program_options)
+if(NOT Boost_FOUND)
+  message(STATUS "Build record-minmax: FAILED (missing Boost::program_options)")
+  return()
+endif(NOT Boost_FOUND)
+
+set(DRIVER "driver/Driver.cpp")
+
+file(GLOB_RECURSE SOURCES "src/*.cpp")
+
+add_executable(record-minmax ${DRIVER} ${SOURCES})
```

Comment:
(optional) how about `circle-minmax` as it's dealing with `circle` files. `circle-record-minmax` seems long...

Reply:
IMHO, `circle-minmax` does not describe what it does by its name..  I would like to go with `record-minmax`. :)
(`circle-record-minmax` is clearer, but it is a bit long as you mentioned)

Reply:
(just a comment)
I would always choose long over ambiguous. What is wrong about the long names, anyway?